eng/common/pipelines/templates/steps/publish-1es-artifact.yml

```diff
@@ -12,10 +12,11 @@ parameters:
   ArtifactPath: ''
   CustomCondition: true
   SbomEnabled: true
+  SkipRenamingArtifact: false
```

Comment:
This file change will be reverted in this PR and I will create a PR in azure-sdk-tools repo later.